go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352
 	golang.org/x/crypto v0.0.0-20211108221036-ceb1ce70b4fa
 	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
-	google.golang.org/protobuf v1.27.1
+	google.golang.org/protobuf v1.28.0
```

Comment:
Could you run `go mod tidy`?

Reply:
[run go mod tidy](https://github.com/micromdm/micromdm/pull/839/commits/551b401dad89259f14760a83f0c7d772a84b7026)

ran go mod tidy